provider/rancher/api.go

```diff
@@ -10,10 +10,12 @@ import (
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/safe"
 	"github.com/containous/traefik/types"
+	"github.com/mitchellh/mapstructure"
 	rancher "github.com/rancher/go-rancher/client"
 )
 
 const labelRancheStackServiceName = "io.rancher.stack_service.name"
+const hostNetwork = "host"
```

Comment:
Can you use a `const` block?

Reply:
Sure thing, on it right now :)

Reply:
Could you please rebase it again to 1.4 branch?